specs/mocks/geolevels.json

```diff
@@ -13,29 +13,29 @@
 }, {
     "id": "fr/canton",
     "name": "French canton",
-    "parents": ["fr/county"]
+    "parents": ["fr/departement"]
 }, {
-    "id": "fr/county",
+    "id": "fr/departement",
     "name": "French county",
     "parents": ["fr/region"]
 }, {
     "id": "fr/district",
```

Comment:
Yes, on the udata side too. we should perform the whole migration because we won't be back on it after I think